oauthenticator/github.py

```diff
@@ -155,6 +155,8 @@ async def authenticate(self, handler, data=None):
         else:
             raise web.HTTPError(500, "Bad response: {}".format(resp_json))
 
+        scopes = resp_json['scope'].split(',')
```

Comment:
This is granted scopes right? Let's be explicit by that and name it so

Reply:
the tests seem to not like `scope` as an attribute of the returned dictionary.  while this https://docs.github.com/en/rest/guides/basics-of-authentication#checking-granted-scopes would suggest otherwise.